src/python/pants_release/generate_github_workflows.py

```diff
@@ -858,16 +840,17 @@ def build_wheels_job(
                     "env": helper.platform_env(),
                 },
                 helper.upload_log_artifacts(name="wheels-and-pex"),
-                *([deploy_to_s3("Deploy wheels to S3")] if for_deploy_ref else []),
                 *(
                     [
                         {
-                            "name": "Upload Pants PEX",
+                            "name": "Upload Wheels and Pex",
```

Comment:
Technically, we don't even need to call `release.py`. We could very well just call `pants package <pants>` and have the wheel.

We don't because `release.py` does some extra special validation.

I think it's worthwhile to move that validation to PR-time, and switch this code just to `package` and upload.

Reply:
Agreed - we shouldn't rely too much on extra scripts when we have a nice build tool :)